README.md

```diff
@@ -7,26 +7,31 @@ Docker powered mini-Heroku. The smallest PaaS implementation you've ever seen.
 
 ## Requirements
 
-- A fresh VM running `Debian 8.2 x64`, `Ubuntu 14.04 x64`, or `Ubuntu 16.04 x64`
+A fresh VM running any one of 
+- `Debian 8.2 x64`
+- `Ubuntu 14.04 x64`
+- `Ubuntu 16.04 x64`
```

Comment:
@MorrisJobke we should probably update the bootstrap script to handle installation on arch.